functions/report_push/push_data_report.py

```diff
@@ -30,8 +34,20 @@ def handler(event, context):
     key = report.get('folder_key')
     run_name = report.get('run_name')
     items = []
-
+    failed_test = 0
     df = wr.s3.read_json(path=['s3://'+qa_bucket+'/allure/' + suite + '/' + key + '/allure-report/history/history-trend.json'])
+    result_df = wr.s3.read_json(path=['s3://'+qa_bucket+'/allure/' + suite + '/' + key + '/allure-report/results/'])
```

Comment:
please use String Formatting
also 
s3.read_json have capability to filter on path_suffix when you're reading files

Reply:
done